vision/automl/edge_container_predict/Dockerfile

```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM tensorflow/serving:1.14.0
+ARG FROM_SUFFIX
```

Comment:
FROM_SUFFIX to VERSION ?
And take 1.14.0 or 1.14.0-gpu as versions?

Reply:
Ah I see.

Other thoughts on this, see:
https://github.com/GoogleCloudPlatform/python-docs-samples/pull/3841

We are using renovate bot which constantly trying to update the versions. Does it make sense to update it to 1.15.0? If so, maybe we should have separate Dockerfiles for cpu and gpu without any ARGs.

WDYT?

Reply:
@genquan9 Also, I think it is better to run the test against the newly built images, and if it succeeds, we can push the new image to gcr.io.